conf/systemd/system/daily-cleanup.timer

```diff
@@ -0,0 +1,18 @@
+# daily-cleanup.timer
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# A daily occuring event to trigger cleanup routines.
+
+
+[Unit]
+Description=Daily Cleanup Timer
+
+[Timer]
+Unit=daily-cleanup.target
+OnCalendar=*-*-* 12:00:00
```

Comment:
Would you like to delay this after boot something like `OnBootSec=10m` as not to clash with the `updater` resuming (as described in `70-kano-updater-download` hook)?

Reply:
Definitely, good catch 👍